pybrake/git.py

```diff
@@ -0,0 +1,47 @@
+import os
+from functools import lru_cache
+
+from .utils import logger
+
+
+@lru_cache(maxsize=1000)
```

Comment:
Is this a Python 3 thing? How is it called?

Reply:
It is function decorator, shortcut for `lru_cache(maxsize=1000)(your_func_name)`.